packages/paste-core/components/checkbox/src/Checkbox.tsx

```diff
@@ -136,7 +162,11 @@ const Checkbox = React.forwardRef<HTMLInputElement, CheckboxProps>(
             </MediaObject>
           </BaseRadioCheckboxLabelText>
         </BaseRadioCheckboxLabel>
-        {helpText && <BaseRadioCheckboxHelpText helpTextId={helpTextId}>{helpText}</BaseRadioCheckboxHelpText>}
+        {helpText && (
+          <BaseRadioCheckboxHelpText element={`${element}_HELP_TEXT`} helpTextId={helpTextId}>
```

Comment:
We're just going with the things unique to Checkbox and Radio button right? Label, Help Text and Error will remain the base so that can be customized as a whole form for each input type